thirdeye/thirdeye-frontend/tests/helpers/resolver.js

```diff
@@ -0,0 +1,11 @@
+import Resolver from '../../resolver';
+import config from '../../config/environment';
```

Comment:
Looks like this is auto generated by ember. The same way  
```
import startApp from '../helpers/start-app';
import destroyApp from '../helpers/destroy-app';
```
is generated for acceptance test. Is this a setting I can modify to change those to absolute path?